xbmc/guilib/GUIListItemLayout.h

```diff
@@ -60,5 +61,7 @@ class CGUIListItemLayout final
 
   INFO::InfoPtr m_condition;
   KODI::GUILIB::GUIINFO::CGUIInfoBool m_isPlaying;
+  unsigned int m_infoUpdateMillis = XbmcThreads::EndTime::InfiniteValue;
```

Comment:
needs to be changed to (untested)
```c
unsigned int m_infoUpdateMillis = std::numeric_limits<std::chrono::milliseconds>::max();
```
see ronie's + jenkins' logs and this commit https://github.com/xbmc/xbmc/pull/19619/commits/decf88520dd660488e4e3d036677dc5a2397596f

Reply:
Ooops.